server/models/UserToken.ts

```diff
@@ -26,12 +26,8 @@ class UserToken extends Model<InferAttributes<UserToken>, InferCreationAttribute
   public declare user?: NonAttribute<typeof models.User>;
   public declare client?: NonAttribute<typeof models.Application>;
 
-  getScope() {
-    return this.scope;
-  }
-
   hasScope(scope) {
-    return this.getScope().includes(scope);
+    return this.scope && this.scope.includes(scope);
```

Comment:
Simpler and support `null` scope.

Reply:
:ice_cream: We can make it safer if we specify the return type

```
hasScope(scope): boolean {
```

```suggestion
    return Boolean(this.scope && this.scope.includes(scope);)
```